sql/core/src/test/resources/sql-tests/inputs/sql-compatibility-functions.sql

```diff
@@ -22,3 +22,12 @@ SELECT string(1, 2);
 -- SPARK-21555: RuntimeReplaceable used in group by
 CREATE TEMPORARY VIEW tempView1 AS VALUES (1, NAMED_STRUCT('col1', 'gamma', 'col2', 'delta')) AS T(id, st);
 SELECT nvl(st.col1, "value"), count(*) FROM from tempView1 GROUP BY nvl(st.col1, "value");
+
+-- Spark-32793: Rewrite AssertTrue with RaiseError
+SELECT assert_true(true), assert_true(boolean(1));
```

Comment:
These tests are related to database compatibility?

Reply:
Whoops, I parked these here on accident. I'll move it to `misc-functions`.